core/linux.go

```diff
@@ -434,10 +434,7 @@ func (l *library) GetStaticLibs(ctx blueprint.ModuleContext) []string {
 	g := getBackend(ctx)
 	libs := []string{}
 	for _, moduleName := range l.Properties.ResolvedStaticLibs {
-		dep, _ := ctx.GetDirectDep(bobName(moduleName))
-		if dep == nil {
```

Comment:
Sorry, I didn't see this one last time around. Do we want this panic too?

Reply:
Yep, reverted